pkg/attestation/verifier/verifier.go

```diff
@@ -49,22 +49,46 @@ func VerifyBundle(ctx context.Context, bundleBytes []byte, tr *TrustedRoot) erro
 		return fmt.Errorf("invalid bundle: %w", err)
 	}
 
+	var signingCert *x509.Certificate
 	if bundle.GetVerificationMaterial() == nil || bundle.GetVerificationMaterial().GetCertificate() == nil {
-		// nothing to verify
-		return ErrMissingVerificationMaterial
-	}
+		// it's a malformed bundle (according to specs) but still verifiable
+		if bundle.GetVerificationMaterial().GetX509CertificateChain() != nil {
```

Comment:
I'd add a todo here to remove it and mark it as deprecated

Reply:
Done.